SICP_book/chapter_2_1.metta

```diff
@@ -130,23 +161,23 @@
 ; (the point whose coordinates are the average of the coordinates of the endpoints). To try your procedures,
 ; you'll need a way to print points
 
-(= (make-segment $start $end) (cons-atom $start $end))
+(= (make-segment $start $end) ($start . $end))
```

Comment:
To be sure `first-pair/second-pair` is applicable to result of `make-segment` `make-pair` should be used here:
```suggestion
(= (make-segment $start $end) (make-pair $start $end))
```